### ISSUE TITLE
Deprecation warning: telnetlib

### ISSUE THREAD

**Issue by bewing** [napalm-automation/napalm#2119]
```
napalm/ios/ios.py:22
  /home/bewing/Projects/napalm/napalm/ios/ios.py:22: DeprecationWarning: 'telnetlib' is deprecated and slated for removal in Python 3.13
    import telnetlib

```

https://github.com/ktbyers/netmiko/issues/3177

Will need to update code once @ktbyers refactors/vendors telnetlib

**Comment by ktbyers** [comment]
See:

https://github.com/napalm-automation/napalm/pull/2137

**Comment by brandan-schmitz** [comment]
I was wondering if there was any plans to get a new release made? It looks like the changes for this were merged into the develop branch back in march and a release has no been created in over a year at this point.

https://github.com/napalm-automation/napalm/pull/2137

**Comment by ktbyers** [comment]
@brandan-schmitz Yes, we should be able to do it now...I was waiting on Juniper PyEZ and them releasing their PyEZ fix of telnetlib (PY313 support).

It looks like they just did the release to support this...

So NAPALM can probably get released fairly soon.

Though I probably can't get to it before the end of the month--due to other work I need to get done.

**Comment by frebib** [comment]
Any chance of a new release that contains this fix? Debian Trixie releases in a couple of weeks and ships Python 3.13. napalm as of it's latest release does not work. It's been over a year

**Comment by ktbyers** [comment]
@frebib I will try to do this some time this week or next week.

**Comment by ktbyers** [comment]
@frebib This is done:

https://github.com/napalm-automation/napalm/releases/tag/5.1.0

NAPALM 5.1.0 is on pypi now.
